utils/unittest/execution_state.go

```diff
@@ -24,7 +24,7 @@ const ServiceAccountPrivateKeySignAlgo = crypto.ECDSAP256
 const ServiceAccountPrivateKeyHashAlgo = hash.SHA2_256
 
 // Pre-calculated state commitment with root account with the above private key
-const GenesisStateCommitmentHex = "8476e9a47da2f993fd0d04efc10238113f217b0c037ccf903613088d51027d6a"
+const GenesisStateCommitmentHex = "689b56c1784938e2089b5c84feef3a6bf3429eaba776473c75583a642d29b628"
```

Comment:
@tarakby in the last PR I had to update the crypto package to v0.24.10 to be compatible with the latest version of flow-go-sdk, is this change of root hash expected? or something else is off?

Reply:
Better to keep it `v0.24.9` and that would exclude the crypto assumption in your state change. But anyway the move from `v0.24.9` to `v0.24.10` should not impact the state.

Reply:
hmm, then we probably have a problem somewhere else.

Reply:
Isn't this expected from deploying the new EVM contract during bootstrapping? Maybe remove the deployment temporarily and see if it still changes

Reply:
bootstraping is now behind a feature flag won't change state by default, it seems to be because of the update of the core contract versions

Reply:
Gregor and I decided to roll back to [3b479f6](https://github.com/onflow/flow-go/pull/4866/commits/3b479f6b0995e6f19fa044cea4c22245db1c8fb2) where everything was okey, and we do a later PR to update the master with latest version of emulator.